internal/proxy/options.go

```diff
@@ -80,11 +84,9 @@ type Options struct {
 	CookieSecure   bool          `envconfig:"COOKIE_SECURE" default:"true"`
 	CookieHTTPOnly bool          `envconfig:"COOKIE_HTTP_ONLY"`
 
-	PassAccessToken bool `envconfig:"PASS_ACCESS_TOKEN" default:"false"`
-
-	// These options allow for other providers besides Google, with potential overrides.
-	Provider string `envconfig:"PROVIDER" default:"google"`
-	Scope    string `envconfig:"SCOPE"`
+	// TODO: What should we do with scope? Right now it doesn't seem to make much sense since SSO is fairly opiniated
+	// in what it needs and asks for from identity providers. Do we need to allow this to overridden?
+	Scope string `envconfig:"SCOPE"`
```

Comment:
Hmm..I'm not sure if it's something that would be used in the short term...maybe at some point in the future if further restructuring is done, in which case it could always be re-added.

But right now, I don't think overriding the scope brings about any benefits (Overriding and _adding_ an additional scope **or** _reducing_ the scopes wouldn't necessarily change anything functionality wise, but could easily cause issues)?

I'm not sure if there's some edge case where it would be useful though 🤔